test-sbt/shared/src/main/scala/zio/test/runner/ZTestFramework.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.test.runner
+
+import sbt.testing._
+import zio.test.DefaultRunnableSpec
+
+final class ZTestFramework extends Framework {
+  override val name = s"${Console.UNDERLINED}ZIO-Test${Console.RESET}"
+
+  val fingerprints: Array[Fingerprint] = Array(RunnableSpecFingerprint)
+
+  override def runner(args: Array[String], remoteArgs: Array[String], testClassLoader: ClassLoader): ZTestRunner =
+    new ZTestRunner(args, remoteArgs, testClassLoader)
+}
+
+object RunnableSpecFingerprint extends SubclassFingerprint {
+  val superclassName: String  = classOf[DefaultRunnableSpec].getCanonicalName
```

Comment:
The `getCanonicalName` may be the problem here, at least with other classes. `getName` is the "reflective" name; `getCanonicalName` is the import statement name.

Reply:
I tested and they return the same string: 
`classOf[RunnableSpec[_,_]].getName` :   zio.test.RunnableSpec
`classOf[RunnableSpec[_,_]].getCanonicalName`: zio.test.RunnableSpec

Reply:
Makes sense, no mangling or otherwise funny business!